packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1790,16 +1790,63 @@ In the final example, end users are required to verify two Authenticators before
+| stepUp  | Defines the authenticators permitted for the secondary authentication step of password recovery | [Self Service Password Reset Action Step-up Requirement object](#self-service-password-reset-action-step-up-requirement-object) | No |
+
+###### Self Service Password Reset Action Primary Requirement object
+
+<ApiLifecycle access="ie" />
+
+Defines the authenticators permitted for the initial authentication step of password recovery
+
+| Property | Description | Data Type | Supported Values | Required | Default |
+| -------- | ----------- | --------- | ---------------- | -------- | ------- |
+| methodConstraints | Specifies an authenticator-specific constraint on the values in the `methods` array. Currently, Google OTP is the only accepted constraint for the `OTP` method. | Array of [Self Service Password Reset Action Primary Requirement Constraint object](#self-service-password-reset-action-primary-requirement-constraint-object) | `[ {"method": "otp", "allowedAuthenticators": [ { "key": "google_otp" } ] } ]` | No | |
+| methods | Authenticator methods allowed for the initial authentication step of password recovery. The `OTP` method requires a constraint limiting it to the Google authenticator. | Array | `PUSH`, `SMS`, `VOICE`, `EMAIL`, `OTP` | No | |
+
+###### Self Service Password Reset Action Primary Requirement Constraint object
+
+<ApiLifecycle access="ie" />
+
+Constraints on the values specified in the `selfServicePasswordReset.requirement.primary.methods` array. Specifying a constraint limits methods to specific authenticators. Currently, Google OTP is the only accepted constraint.
+
+| Property | Description | Data Type | Supported Values | Required | Default |
+| -------- | ----------- | --------- | ---------------- | -------- | ------- |
+| allowedAuthenticators | Limits the authenticators that can be used for a given method. Currently, only the `OTP` method supports constraints and Google authenticator is the only allowed authenticator. | Array of [authenticator keys](/docs/reference/api/authenticators-admin/#authenticator-properties). | `[ { "key": "google_otp" } ]` | No | |
+| method  | Specifies the method that is limited to the specific authenticator in `allowedAuthenticators`. Currently, Google OTP is the only accepted constraint. | String | `OTP` | No | |
+
+###### Self Service Password Reset Action Step-up Requirement object
+
+<ApiLifecycle access="ie" />
+
+Defines the secondary authenticators needed for password reset if `selfServicePasswordReset.requirement.stepUp.required` is true. The following are three valid configurations:
+
+* `required` = false
+* `required` = true with no methods to use any SSO authenticator
+* `required` = true with `security_question` as the method
+
+| Property | Description | Data Type | Supported Values | Required | Default |
```

Comment:
Is there a default value for these rows? Maybe we don't need this column?

Reply:
Nope, no default values. I'll remove this column in another commit. Thanks for the suggestion!